tests/sqlparser_hive.rs

```diff
@@ -276,6 +276,18 @@ fn parse_create_function() {
     );
 }
 
+#[test]
+fn filtering_during_aggregation() {
+    let rename = "SELECT array_agg(name) FILTER (WHERE name IS NOT NULL) FROM region";
+    println!("{}", hive().verified_stmt(rename));
+}
+
+#[test]
+fn filter_as_alias() {
+    let rename = "SELECT name filter FROM region";
```

Comment:
we need a different style test - this fails because it is re-written as `SELECT name AS filter FROM region`